### ISSUE TITLE
Html tags get renderd

### ISSUE THREAD

**Issue by mellester** [microsoft/vscode-copilot-release#196]
I asked the question.
1.  can you let the image float to the left of the <P>
The `<P>` does not get displayed in the chat history. Question does get answered so great job.

**Comment by roblourens** [comment]
Yeah, currently requests are rendered as markdown, that may change https://github.com/microsoft/vscode-copilot-release/issues/107
